lib/neuron/fragment.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Neuron.Fragment do
+  alias Neuron.Store
+
+  @doc ~S"""
+  registers a fragment that will automatically be added to future mutations and queries that require it
+
+  ## Example
+
+  ```elixir
+  Neuron.register("\"\"
```

Comment:
Could we run this as a `DocTest`? This way we make sure we don't forget to update it if we change the API.